test/v3/amendPeriodSummary/AmendPeriodSummaryServiceSpec.scala

```diff
@@ -46,8 +38,6 @@ class AmendPeriodSummaryServiceSpec extends ServiceSpec {
 
   private val periodIncomeWithCl290Enabled = Def2_Amend_PeriodIncome(turnover = Some(2000.00), None, taxTakenOffTradingIncome = Some(2000.00))
 
-  private val periodIncomeWithCl290Disabled = Def1_Amend_PeriodIncome(turnover = Some(2000.00), None)
```

Comment:
This can be changed to use Def2_Amend_PeriodIncome and will need None to be passed as the parameter for taxTakenTradingIncome